beacon-chain/blockchain/execution_engine.go

```diff
@@ -294,6 +303,14 @@ func (s *Service) getPayloadAttribute(ctx context.Context, st state.BeaconState,
 			return false, emptyAttri
 		}
 	}
+	if e > stateEpoch {
+		val, ok = s.trackedProposer(st, slot)
```

Comment:
Would it be more appropriate to include `features.Get().PrepareAllPayloads` within `s.trackedProposer` to ensure it consistently returns true? This approach seems to offer a cleaner abstraction